test/com/tw/go/plugin/AbstractGitHelperTest.java

```diff
@@ -259,6 +260,33 @@ public void shouldInitAddCommit() throws Exception {
         assertThat(revision.getModifiedFiles().get(0).getAction(), is("added"));
     }
 
+    @Test
+    public void shouldFetchWithDefaultDepth() {
+        GitConfig config = new GitConfig("https://github.com/mdaliejaz/samplerepo.git");
```

Comment:
I was going to do this -- but I saw the shallow clone tests were `@Ignored` and they didn't work, so I presumed the whole shallow-cloning a local repo didn't work? I'm happy to re-enable?